utils/zero_boot/service_creator/README.md

```diff
@@ -0,0 +1,91 @@
+# Service creator tool
```

Comment:
What is this file doing here ?
I think you can remove the full docs/zero_boot directory